### DISCUSSION TITLE
Full unenrollment

### DISCUSSION THREAD

**Discussion by ericshu6** [3kh0/ext-remover#283] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
there is a method for full unenrollment
1. unscrew the battery for 30 sec
2. unenrolled gg ez
there may be extra steps that you need to do but this is the basic idea

**Comment by 3kh0** [comment]
This also works, but this method is faster,

1. Throw your chromebook down a 50 foot hill.

**Reply by Fallenmoon8080** [reply]
Hmmm. After much and great deliberation, I believe this method to be hardly falsified, and comdemn anyone who uses it to a broken chromebook🤓

**Reply by Theblackmamba824** [reply]
> This also works, but this method is faster,
> 
> 1. Throw your chromebook down a 50 foot hill.

tho an even faster method would be to take Ur Chromebook apart pour about a gallon of Scripto lighter fluid on it then take a sondiko blow torch lighter... specifically those brands then torch the inside of the Chromebook then attempt to put if back together and



BAMMMM THERE U HAVE IT A UNEROLLED SCHOOL CHROMEBOOK

**Reply by SomePVZPlayerMatt** [reply]
Instructions unclear, my chromebook broke 🤓

**Reply by skoopy-star** [reply]
this works everybody should do this RN

**Reply by Aka-but-nice** [reply]
It's way easier to just grab your Glock 18 and unenroll it from life!

**Reply by winghf** [reply]
Dang i should have took my chromebook with me when i went hiking dAng it.

**Reply by Aka-but-nice** [reply]
yea same

**Comment by rwg100** [comment]
Depending on the type of chromebook, the battery may not be able to be unscrewed. I have found a quick and easy solution though, for this case is only for chromebooks in which you cannot unscrew it.
The steps are:

1. Open up the back cover
2. Locate the battery, it should look silver with orange and blue tape
3. Carefully unscrew the battery
4. Wait 30 seconds
5. Put the battery back in
6. Take your Chromebook to a spot in your house with weak Wi-Fi connection
7. Connect a USB to your chromebook (any type works, even if it already has data
8. Powerwash your chromebook
9. Take the chromebook to the spot next to your router
10. Disconnect the wifi
11. Nuke the chromebook

**Reply by Theblackmamba824** [reply]
accurate results

**Reply by 3kh0** [reply]
it works

**Reply by rwg100** [reply]
"how do you know?"

"trust me bro"

**Reply by 3kh0** [reply]
try https://coolelectronics.me/unroll

**Comment by 3kh0** [comment]
Also https://coolelectronics.me/unroll works well

**Reply by Aka-but-nice** [reply]
Refer to #348 #347
